packages/neutron-sdk/src/interchain_queries/register_queries.rs

```diff
@@ -24,14 +26,18 @@ fn register_interchain_query_msg(
     transactions_filter: String,
     update_period: u64,
 ) -> NeutronResult<NeutronMsg> {
-    let register_msg = NeutronMsg::register_interchain_query(
-        query_type.into(),
-        kv_keys,
-        transactions_filter,
-        connection_id,
-        update_period,
-    );
-    Ok(register_msg)
+    match query_type {
+        QueryType::KV => Ok(NeutronMsg::register_interchain_query(
```

Comment:
i think that the naming here is very confusing. `register_interchain_query` looks like an action, but it simply returns a message